libz-rs-sys-cdylib/Cargo.toml

```diff
@@ -14,6 +14,12 @@ rust-version = "1.75" # MSRV
 name = "z_rs" # turns into e.g. `libz_rs.so`
 crate-type=["cdylib"]
 
+[profile.dev]
+panic = "abort" # abort on panics. This is crucial, unwinding would cause UB!
+
+[profile.release]
+panic = "abort" # abort on panics. This is crucial, unwinding would cause UB!
```

Comment:
Can you add `#[cfg(not(panic = "abort")] compile_error!()` to `libz-rs-sys-cdylib/src/lib.rs`?